### ISSUE TITLE
Conduct work session on Moped tags, categories, and project funding

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#5465]
[List of staff-sourced taxonomies](https://docs.google.com/spreadsheets/d/1VGO57EL1TtASbLz2yeWyLwcTB1lT9LR8G6_QZ4WSifs/edit?usp=sharing)

- Funding sources
- Project types
- Project tags
- Project groups 
- Project sponsor
- Programs?
- ...What's missing?

[Notes](https://docs.google.com/document/d/1xMTBL-uyNH9keUydl7K29liZdIIHBMdiYTJuqaylTpE/edit#)

**Comment by amenity** [comment]
Per discussion in Slack, we likely don’t need a meeting. @zappfy, will these all be staff sourced taxonomies? Assuming we can start out with whatever is in Access, can you pull those lists and attach here as CSVs?

**Comment by zappfyllc** [comment]
Just a reminder that the current IMPD taxonomies for these additional project domains/attributes are [linked above in the initial post](https://docs.google.com/spreadsheets/d/1VGO57EL1TtASbLz2yeWyLwcTB1lT9LR8G6_QZ4WSifs/edit#gid=0)

**Comment by zappfyllc** [comment]
re: `CorridorPlan` values stored in the CurrentPhase column in IMPD:

We made the decision to not include CorridorPlan in our initial list of phases/sub-phases/milestones/statuses because of feedback from the card-sorting activity and our initial guesses at whether that actual fit the domains listed. Instead, we would propose that CorridorPlan might be a `Project Type` or some other attribute listed in this issue where we can track projects sourced from TXDOT or the CPO (I believe I'm using that correctly, apologies if not). This way, the projects can still be tracked in Moped, but we don't design around the exception since the assumption is that most projects in Moped do not go through this phase/stage (hence the reason we got such dominant feedback from the exercise that the phase was irrelevant or didn't make sense).
